pkg/distro/defs/fedora/distro.yaml

```diff
@@ -12,6 +12,18 @@
       - "geolite2-country"
       - "plymouth"
 
+  cloud_init_services: &cloud_init_services
+    - "cloud-init.service"
+    - "cloud-config.service"
+    - "cloud-final.service"
+    - "cloud-init-local.service"
+
+  cloud_init_services_new: &cloud_init_services_new
```

Comment:
(super nitpick) maybe `cloud_init_service_f43plus` ? or a comment for both? Its nice in the place where it is used but here its not obvious what new/old means

Reply:
Yeah, bad at names.  F43plus makes more sense